dashboard/dashboard-front/src/components/jobs/JobStatus.vue

```diff
@@ -1,12 +1,29 @@
 <script setup lang="ts">
 import { type Ref, computed } from 'vue'
 
-const props = defineProps(['status'])
-const status: Ref<string | null> = computed(() => props.status)
+const props = defineProps({
+    status: { type: String, required: false },
+    short: { type: Boolean, required: false, default: false },
+})
+const status: Ref<string> = computed(() => props.status || "")
+const color: Ref<string> = computed(() => {
+    if (status.value == "running") {
+        return "green"
+    } else if (status.value == "error") {
+        return "red"
+    } else {
+        return "orange"
+    }
```

Comment:
Generally, it is preferred to use strict equality over the normal one.

```suggestion
    if (status.value === "running") {
        return "green"
    } else if (status.value === "error") {
        return "red"
    } else {
        return "orange"
    }
```

Reply:
Or it could be changed to `switch case`